src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs

```diff
@@ -123,6 +123,7 @@ public override IEnumerable<DownloadClientItem> GetItems()
                 else if (torrent.IsFinished && torrent.State != DelugeTorrentStatus.Checking)
                 {
                     item.Status = DownloadItemStatus.Completed;
+                    item.CanBeDeleted = true;
```

Comment:
Nope, finished seeding is determined by the flags later. 'IsFinished' simply means finished downloading.    
Basically, only Diskstation can have a different value for CanBeDeleted vs CanBeRemovedFromClient.

So for all other clients CanBeDeleted = CanBeRemovedFromClient.